README.md

```diff
@@ -18,20 +18,25 @@ Allows to create a gulp files in classes, each method of which can be a gulp tas
     
     ```typescript
     import {Gulpclass, Task, SequenceTask} from "gulpclass";
-
-    let gulp = require("gulp");
-    let del = require("del");
+    import * as gulp from "gulp";
+    import * as del from "del";

```

Comment:
If you want to be friendly we can always add a notice about this topic somewhere in the README.